src/components/Dialog/TextDialog/QuestionDialog.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { userEvent, within, expect, fn, waitFor } from "@storybook/test";
+
+import { Meta, StoryObj } from "@storybook/vue3";
+import QuestionDialog from "./QuestionDialog.vue";
+import { UnreachableError } from "@/type/utility";
+
+const meta: Meta<typeof QuestionDialog> = {
+  component: QuestionDialog,
+  args: {
+    type: "info",
+    modelValue: true,
+    title: "タイトル",
+    message: "メッセージ",
+    buttons: ["A", "B", "C"],
+
+    onOk: fn(),
+    "onUpdate:modelValue": fn(),
```

Comment:
これ調べた感じquasarのコンポーザブルにhideのemitがあるっぽいので、こうする手もあるかも。
```suggestion
    onHide: fn(),
```
（こうすると
```ts
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  "update:modelValue": (val: boolean) => true,
```
のとこが不要になってちょっときれい、くらいですが。